pom.xml

```diff
@@ -51,6 +53,17 @@
       <scope>provided</scope>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
+      <version>2.0.2-beta</version>
+    </dependency>
```

Comment:
PowerMockito is more trouble than it is worth. We don't need PowerMockito, trust me (we didn't need it for 92% line coverage for a game running on a badly written virtual reality port of an alpha version of a java game engine)